Automattic/newspack-plugin
# add/donations mailing

### All Submissions:






### Changes proposed in this Pull Request:



Adds ability to edit and send and email after a reader has donated, if the site uses Stripe platform.
The UI is very minimal, basically just a link to edit the email:

<img width="992" alt="image" src="https://user-images.githubusercontent.com/7383192/137296052-2d831eed-13d0-419a-ba28-1aa6fc63634e.png">

In the future, it might display the number of sent emails and the email preview.

The other part is the email itself - it's very simple:

<img width="561" alt="image" src="https://user-images.githubusercontent.com/7383192/137338930-99b38d08-813f-4989-991d-534c40f0e84e.png">

### How to test the changes in this Pull Request:

1. Use latest `newspack-newsletters` (with https://github.com/Automattic/newspack-newsletters/pull/617)
2. Configure your instance to use Stripe as the Reader Revenue platform
3. Visit the "Emails" tab of the Reader Revenue wizard
4. Observe a single UI element with a link to edit the "Thank you!" email, but don't edit it yet!
5. Make a donation, observe an email with donation details sent to the donor's email address*
6. Edit the email, make a donation again, observe the updated version of the email is sent
7. Deactivate the Newspack Newsletters plugin, observe the email is not sent and the "Emails" tab features a plugin activation prompt
8. After activating the plugin, wizard should refresh with the previous UI

\* WP has to be configured to send emails, e.g. [Local](https://localwp.com/) has MailHog pre-configured

### Other information:
